src/types/CodepointWidthDetector.cpp

```diff
@@ -147,17 +156,21 @@ namespace
         UnicodeRange{ 0x22bf, 0x22bf, CodepointWidth::Ambiguous },
         UnicodeRange{ 0x2312, 0x2312, CodepointWidth::Ambiguous },
         UnicodeRange{ 0x231a, 0x231b, CodepointWidth::Wide },
-        UnicodeRange{ 0x2329, 0x232a, CodepointWidth::Wide },
-        UnicodeRange{ 0x23e9, 0x23ec, CodepointWidth::Wide },
-        UnicodeRange{ 0x23f0, 0x23f0, CodepointWidth::Wide },
-        UnicodeRange{ 0x23f3, 0x23f3, CodepointWidth::Wide },
-        UnicodeRange{ 0x2460, 0x24e9, CodepointWidth::Ambiguous },
+        UnicodeRange{ 0x2328, 0x232a, CodepointWidth::Wide },

```

Comment:
If possible -- and I know this is a lot of work -- i'd like a comment on each line that refers to a footnote about us overriding these. Just so we can come find it later.